assets/javascripts/modules/analytics/setup.js

```diff
@@ -1,34 +1,37 @@
-/*global guardian:true */
 define([
     'utils/cookie',
     'modules/analytics/ga',
     'modules/analytics/omniture',
+    'modules/analytics/remarketing',
     'modules/analytics/krux',
     'modules/analytics/affectv'
 ], function (
     cookie,
     ga,
     omniture,
+    remarketing,
     krux,
     affectv
 ) {
     'use strict';
 
-    function init() {
-
-        var analyticsEnabled = (
-            guardian.analyticsEnabled &&
-            !navigator.doNotTrack &&
-            !cookie.getCookie('ANALYTICS_OFF_KEY')
-        );
+    var analyticsEnabled = (
+        window.guardian.analyticsEnabled &&
+        !navigator.doNotTrack &&
+        !cookie.getCookie('ANALYTICS_OFF_KEY')
+    );
 
+    function init() {
         if (analyticsEnabled) {
             ga.init();
             omniture.init();
+        }
+
+        if (analyticsEnabled && !window.guardian.isDev) {
```

Comment:
I would probably put the `if (!window.guardian.isDev)` condition inside the `if (analyticsEnabled)` block